frontend/pages/packs/EditPackPage/EditPackPage.tsx

```diff
@@ -122,7 +121,7 @@ const EditPacksPage = ({
     : [];
 
   const onCancelEditPack = () => {
-    return dispatch(push(PATHS.MANAGE_PACKS));
+    return router.push(PATHS.MANAGE_PACKS);
```

Comment:
Where possible, let's remove `dispatch` from the component altogether. I know in many cases we can't because of `FlashMessage` but in this component it looks like `dispatch` is no longer being used. That means we can remove redux completely from this component! Maybe happening elsewhere?

Reply:
I looked the `EditPackPage` component but I still see `dispatch` being used for the flash message. Am I looking at the wrong  file?